src/aui-form-validator/tests/unit/js/tests.js

```diff
@@ -190,6 +190,48 @@ YUI.add('aui-form-validator-tests', function(Y) {
             Y.Assert.isFalse(validator.hasErrors());
         },
 
+        /*
+         * Check if validator correctly validates dates in pt-br format dd/MM/yyyy
+         * @tests AUI-1940
+         */
+          'test date format': function() {
```

Comment:
Hi @thiago-rocha, the test case that failed is not regarding the date format  and is not caused by this commit so I will remove it and treat this case in another lps.